src/Mod/Draft/draftfunctions/move.py

```diff
@@ -160,3 +162,46 @@ def move(objectslist, vector, copy=False):
         gui_utils.select(newobjlist)
     if len(newobjlist) == 1: return newobjlist[0]
     return newobjlist
+
+
+#   Following functions are needed for SubObjects modifiers
+#   implemented by Dion Moult during 0.19 dev cycle (works only with Draft Wire)
+
+
+def move_vertex(object, vertex_index, vector):
+    points = object.Points
```

Comment:
The comment should be added to the docstring of the function, that way the information stays together.